OBJECT/SetObjectTargettableByPlayer.md

```diff
@@ -0,0 +1,19 @@
+---
+ns: OBJECT
+---
+
+```c
+// 0xB39F03368DB0CAA2
+void _SET_OBJECT_TARGETTABLE_BY_PLAYER(Object object, BOOL setFlag34, BOOL setFlag35);
+```
+Sets the 34th and 35th object flags related to player peds.

```

Comment:
Is there any more context for what these flags are?

Reply:
No tried to search on decompiled scripts the only thing i found is 
_SET_OBJECT_TARGETTABLE_BY_PLAYER is called with setFlag34 to false

Maybe it's about homing missiles? (since SET_OBJECT_TARGETTABLE is here and it's for homing missiles)